pkg/ds/daemon_set.go

```diff
@@ -300,24 +303,41 @@ func (ds *daemonSet) WatchDesires(
 		ds.publishStatus(ctx)
 	}()
 
+	nodesToAdd := make(chan []types.NodeName, 100)
```

Comment:
This could be 10x as big without causing problems right?

Reply:
yeah I imagine so

Reply:
i'll comment it too